### DISCUSSION TITLE
Expose Store Parameters to XML

### DISCUSSION THREAD

**Discussion by Jlobblet** [FakeFishGames/Barotrauma#5460] [Ideas and suggestions]
Currently, the store parameters are all hardcoded. It would be nice if they were exposed to XML for modding (able to be modified by each outpost type ideally)

https://github.com/Regalis11/Barotrauma/blob/0697d7fc6412f972e5089fa88b693b486e55fb86/Barotrauma/BarotraumaShared/SharedSource/Map/Map/Location.cs#L87-L99

**Comment by ekqvist** [comment]
I've exposed multiple store parameters now and made them possible to be defined for every location type. They will default to the old values if not defined. Of course, not yet publicly available, but probably coming later on.
